### DISCUSSION TITLE
Support for PDF export

### DISCUSSION THREAD

**Discussion by zeelmodi-simform** [dolanmiu/docx#2560] [Q&A]
Is it possible to generate word formatted document as PDF? I'm using this package to export as a docx format. But I would also like to export as PDF as well instead of manually converting it. Any help would be appreciated.

**Comment by dolanmiu** [comment]
Not supported nope, but there are services you can use like so: https://www.ilovepdf.com/word_to_pdf

**Reply by zeelmodi-simform** [reply]
Hey @dolanmiu , Is it possible to directly generate a pdf using the code used for generating docx? Can you please suggest any package?
